x-pack/plugins/security/public/account_management/index.ts

```diff
@@ -6,3 +6,6 @@
  */
 
 export { accountManagementApp } from './account_management_app';
+
+export type { ChangePasswordProps } from './change_password/change_password';
+export type { PersonalInfoProps } from './personal_info/personal_info';
```

Comment:
__nit:__ please export these types from `./change_password | personal_info/index.ts`. We use `index.ts` as a grouping mechanism for all exports where feasible.

```suggestion
export type { ChangePasswordProps } from './change_password';
export type { PersonalInfoProps } from './personal_info';
```

Reply:
Got it, thanks! Updated here: https://github.com/elastic/kibana/pull/99791/commits/9b932dcf9ba0c4a2a65fe5ecd792f3db93cfebab